content/assignments/mini-libc/tests/memory/malloc.c

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#include <stdlib.h>
+#include <unistd.h>
+
+#define		ALLOC_SIZE	(4*1024*1024)
+#define		INFINITE	1000000
```

Comment:
```suggestion
#define		ALLOC_SIZE	(4 * 1024 * 1024)
#define		INFINITE	1000000
```
Since you duplicate these `#defines` so much, perhaps you should place them in a separate `.h` file and `#include` it. Same for the other files that contain this code.

Reply:
@teodutu, I will leave this for later. There are multiple places that can be improved by refactoring common code in test files (particularly shell scripts), so we will do it later. I created issue #226 for this.